installer/install.iss

```diff
@@ -996,16 +996,8 @@ end;
 function InitializeSetup:Boolean;
 var
     CurrentVersion,Msg:String;
-    Version:TWindowsVersion;
     ErrorCode:Integer;
 begin
-    GetWindowsVersionEx(Version);
-    if (Version.Major<6) then begin
```

Comment:
As discussed in https://github.com/git-for-windows/build-extra/pull/327#discussion_r571476967, this check is no longer necessary since the minimum version required by InnoSetup is now also Windows Vista (see `MinVersion` above).